src/Sinch/Conversation/Hooks/ChannelEventAllOfChannelEventNotification.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Sinch.Conversation.Hooks
+{
+    /// <summary>
+    ///     ChannelEventAllOfChannelEventNotification
+    /// </summary>
+    public sealed class ChannelEventAllOfChannelEventNotification
+    {
+
+        /// <summary>
+        /// Gets or Sets Channel
+        /// </summary>
+        [JsonPropertyName("channel")]
+        public ConversationChannel? Channel { get; set; }
+
+        /// <summary>
+        ///     The type of event being reported.
+        /// </summary>
+        [JsonPropertyName("event_type")]
+        public string EventType { get; set; }
+        
+
+        /// <summary>
+        ///     An object containing additional information regarding the event. The contents of the object depend on the channel and the event_type.
+        /// </summary>
+        [JsonPropertyName("additional_data")]
+        public Object AdditionalData { get; set; }
```

Comment:
Is this enabling to handle the received content withtou dedicated action from end user? 
A map of <String, String> (or even more complex hierarchy) can be received (not yet tested webhooks, myself so not sure and it is an open question)

Reply:
I'll put `JsonObject` here which would allow to easier handle unstructured data